packages/plugin-webpack/plugin.js

```diff
@@ -99,6 +91,7 @@ module.exports = function plugin(config, args) {
           rules: [
             {
               test: /\.js$/,
+              exclude: /node_modules/,
```

Comment:
Only core-js & regenerator-runtime should be loaded directly from node_modules (everything else comes from web_modules). Those CJS files can't pass them through this ESM-only Babel build step without causing issues, so mark them as safe to exclude from the Babel build

Reply:
WFM. In general `exclude: /node_modules/` is a pretty good thing to do with webpack